atlasdb-cassandra/src/test/java/com/palantir/atlasdb/cassandra/CassandraAtlasDbFactoryTest.java

```diff
@@ -1,178 +0,0 @@
-/*
```

Comment:
Turns out all these tests were just for config management, so they've been dispersed to the relevant places (either RefreshableWithInitialDefault or CassandraKvsConfigs)